backend/app/routers/alerts.py

```
@@ -110,10 +113,39 @@ async def create_alert(req: CreateAlertRequest) -> AlertResponse:
+                    parsed = future.result(timeout=NLP_PARSE_TIMEOUT_SECONDS)
+                    # Extract action from parsed NLP result and use it as behavior
+                    if parsed.get("action"):
+                        rule_dict["behavior"] = parsed.get("action")
+                except FutureTimeoutError:
+                    logger.warning(
+                        "NLP parsing timeout after {} seconds while creating alert name={} nl_len={}",
+                        NLP_PARSE_TIMEOUT_SECONDS,
+                        req.name,
+                        len(req.nl or ""),
+                    )
+                except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)